### PR TITLE
Modify Readme and add apps

### PR DESCRIPTION
**Pull request by tudorikas**
-modify Readme
-modify image diagram
-add stream capture
-add alert telegram bot

### CODE REVIEW

File: Classify/classify-image.py [hunk 1]
```diff
@@ -74,22 +74,26 @@ def main(argv):
                     res = runner.classify(features)
 
                     if "classification" in res["result"].keys():
-                        print('Result (%d ms.) ' % (res['timing']['dsp'] + res['timing']['classification']), end='')
+                        #print('Result (%d ms.) ' % (res['timing']['dsp'] + res['timing']['classification']), end='')
```
Comment (krasi-georgiev) [inline Classify/classify-image.py]: can we have some structured logging instead of print statements?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 ![Workflow](img/workflow.png)
 
+Introduction
+============
+
+Aribbada Smart Camera - is a project which aims to perform with the help of artificial intelligence the detection (during the day and at night) and alerting people when a wild animal appears in the area of interest
```
Comment (krasi-georgiev) [inline README.md]:
```suggestion
This project aims to reduce Human-wildlife conflict with the help of video surveillance and artificial intelligence.
The idea is to use a camera for 24h video surveillance and machine learning to detect and alert for animals that pose a threat livelihood or safety of the people in the area.

```

File: Upload/uploadTelegram.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    # Sort list of files based on last modification time in ascending order
+    list_of_files = sorted( list_of_files,
+                            key = lambda x: os.path.getmtime(os.path.join(dir_name, x)))
+    # Iterate over sorted list of files and print file path
+    # along with last modification time of file
+
+    for file in list_of_files:
+        bot.send_photo(channel_id,photo=open(os.path.join(dir_name, file),"rb"))
+        time.sleep(3)
+        os.remove(os.path.join(dir_name, file))
```
Comment (krasi-georgiev) [inline Upload/uploadTelegram.py]: If `send_photo` has some error doesn't that mean the file will not be sent, but will be removed? not sure how is the error handling here?

**Review by krasi-georgiev — COMMENTED** [review]
When we use configurations can we have rtsp and picamera in a single app so that  when `rtsp` config is empty it reads from the picamera otherwise it reads from the rtsp stream

**Comment by tudorikas** [comment]
Will do

**Pull request merged**